scripts/release/version.py

```diff
@@ -1,7 +1,63 @@
+import os
+from enum import StrEnum
+
 import semver
 from git import Commit, Repo, TagReference
 
-from scripts.release.changelog import ChangeKind
+from scripts.release.changelog import (
+    DEFAULT_CHANGELOG_PATH,
+    ChangeEntry,
+    ChangeKind,
+    get_changelog_entries,
+)
+
+COMMIT_SHA_LENGTH = 8
+
+class Environment(StrEnum):
+    DEV = "dev"
+    STAGING = "staging"
+    PROD = "prod"
```

Comment:
nit: instead of calling it, `prod` should we call it `release`? The reason is, the word `prod` is generally used when we actually deploy things to production environment, but in case our case we are not actually deploying things to production environment. We are just releasing them. That's why I think calling it release would be a good idea.

Reply:
this will be changed to `BuildScenario` to better match what Julien is doing. `prod` will be changed to `release`